packages/contracts/test/README.md

```diff
@@ -19,206 +19,3 @@ Only run tests in a given file:
 npx hardhat test test/consent.js
 
 ```
-
-# Testing OFT20Rewards
```

Comment:
Wanted to move this into our rewards package where all the Layer zero rewards sit, otherwise this part of the README doesn't work on this contracts package. Open to keeping a copy here as well though.